### PR TITLE
Remove settings.js

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

`settings.js` being loaded from source in our templates can cause some problems for our CSP.  Additionally, it is something that [isn't really used anymore](https://github.com/department-of-veterans-affairs/va.gov-team/issues/1973)

This PR must be merged after #12254 since that removes the RateLimiter component.

## Testing done

Local testing.  I'm able to get a successful build and can browse localhost without any apparent issues.

## Screenshots

Settings.js being loaded and causing a CSP error

image

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by rianfowler — APPROVED** [review]
fyi @ncksllvn we're removing the settings.js file

**Review by ncksllvn — APPROVED** [review]
Love it!

File: config/webpack.config.js [hunk 1]
```diff
@@ -58,29 +58,6 @@ function getEntryPoints(entry) {
   return getWebpackEntryPoints(manifestsToBuild);
 }
 
-/**
- * The dev server requires settings.js for building the
- * redirects. This loads the settings for use in the watch commands.
- *
- * @return {Object} settings
- */
-function getDevServerSettings() {
```
[the remark this answers is not in the document]
Comment (cvalarida) [reply config/webpack.config.js:67]: This doesn't have to do with `settings.js`. This is the `webpack-dev-server` settings, if I'm not mistaken.

What happens if this is removed?

**Review by cvalarida — APPROVED** [review]

**Comment by cvalarida** [comment]
Closes https://github.com/department-of-veterans-affairs/vets-website/pull/12264

**Comment by bkjohnson** [comment]
@cvalarida err - you're right! merging this PR does close itself!

**Comment by cvalarida** [comment]
:facepalm: Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/8067

**Comment by cvalarida** [comment]
Closes https://github.com/department-of-veterans-affairs/va.gov-team/issues/7946
Third time's the charm, right? :rage3:

**Pull request merged**